packages/@react-aria/dialog/src/useDialog.ts

```diff
@@ -52,13 +52,15 @@ export function useDialog(props: AriaDialogProps, ref: RefObject<HTMLElement>):
     }
   }, [ref]);
 
+  // We do not use aria-modal due to a Safari bug which forces the first focusable element to be focused
+  // on mount when inside an iframe, no matter which element we programmatically focus.
+  // See https://bugs.webkit.org/show_bug.cgi?id=211934.
```

Comment:
Should we mention that we are explicitly setting `aria-hidden` on siblings of the dialog, so that the Dialog still behaves as a modal?